cache-config/testing/ort-tests/t3c-apply-unset-update_test.go

```diff
@@ -26,48 +25,65 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
+const cacheHostName = `atlanta-edge-03`
```

Comment:
Making a global default like this seems like a good idea, but the name "cacheHostName" doesn't really convey enough information for something globally available. It'd also make it a little more obvious it isn't local if it were capitalized, especially since it used to be. Maybe something like `DefaultCacheHostName`?

Reply:
It also took some digging to find this variable. It'd be nice if it were in a more generic location. But I'm not sure we have a file for that today

Reply:
The name was chosen because that was the most common name in its various definitions. I'm not attached to it. I can group stuff like this in a `defaults_test.go`, I think.

Reply:
`t3c-testlibs.go` would have been a place, but you deleted that 😂 
Personally, I like a file named after the directory/package for this kind of thing, e.g. `ort-tests.go` or `ort-tests_test.go`. I find that the most obvious. But `defaults_test.go` is fine too.